### ISSUE TITLE
Improve inputs in ACP setting page

### ISSUE THREAD

**Issue by Eldenroot** [mybbgroup/iPatrol#17]
Here for this field:

![image](https://user-images.githubusercontent.com/778890/65180078-5fc36c80-da5c-11e9-81b5-9751b2fee4d4.png)

**One word per line would be much better as I had suggested before.  Much easier for import/export.**

Same for:
![image](https://user-images.githubusercontent.com/778890/65180121-7669c380-da5c-11e9-8a3a-d42c62e061c7.png)

**Comment by effone** [comment]
I guess comma separated user ids are fine.
It is being used that way in many plugins for ages.

String list can be modified, yes.

**Comment by WildcardSearch** [comment]
I have to agree with @effone here. IDs are fine as comma-separated list. With the strings, I agree with @Eldenroot because what if the string needs to include a comma?

**Comment by Eldenroot** [comment]
OK, leave user IDs as it is now but lets discuss about the string list; one word per line and you can use whatever you want (including comma).
